warehouse/warehouse/scripts/fix_mapfile_paths.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [ $# -lt 1 ] || [ $1 == "-h" ]; then
+    echo "Usage: fix_mapfile_paths.sh mapfile_full_path"
+    exit 0
+fi
+
+mapfile=$1
+
+if ! [ -f $mapfile ]; then
+    echo "ERROR: File not found: $mapfile"
+    exit 1
+fi
+
+# WARNING: HARDCODED STRINGS
+src_pattern="e3sm\/warehouse"
```

Comment:
Please remove these, we need to be able to move things from/to anywhere, not only so we can test it, but also so we can run at other facilities that dont have the same directory layout as acme1.